brew-publish.sh

```diff
@@ -72,6 +72,8 @@ class Rig < Formula
   depends_on "docker-machine"
   depends_on "docker-compose"
   depends_on "docker-machine-nfs"
+  depends_on "unison"
+  depends_on "eugenmayer/dockersync/unox"
```

Comment:
It's not really dockersync, it is just a utility that does the fswatching that he put in a brew tap called dockersync. We are not actually using dockersync